docs/training/lerobot_guide.rst

```diff
@@ -1,6 +1,6 @@
-==========================
-LeRobot X Aloha User Guide
-==========================
+=====================================
+LeRobot X Aloha Stationary User Guide
+=====================================
```

Comment:
The first heading in the document will be the header for the whole document. I think keeping the original heading and adding a subheading for the stationary is best here.